@xen-orchestra/web/src/components/site/pools/PoolSidePanel.vue

```diff
@@ -0,0 +1,243 @@
+        </UiCardTitle>
+        <div class="content">
+          <!-- Pool -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ t('pool') }}</template>
+            <template #value>
+              <UiLink
+                v-if="server.poolId !== undefined && server.poolNameLabel !== undefined"
+                :icon="faCity"
+                size="small"
+                :to="`/pool/${server.poolId}/`"
+              >
+                {{ server.poolNameLabel }}
+              </UiLink>
+            </template>
+            <template v-if="server.poolId !== undefined" #addons>
+              <VtsCopyButton :value="server.poolId" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- ID -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ t('id') }}</template>
+            <template #value>{{ server.id }}</template>
+            <template #addons>
+              <VtsCopyButton :value="server.id" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- Description -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ t('description') }}</template>
+            <template #value>{{ server.poolNameDescription }}</template>
+            <template v-if="server.poolNameDescription !== undefined" #addons>
+              <VtsCopyButton :value="server.poolNameDescription" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- tag -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ t('tags') }}</template>
+            <template #value>
+              <UiTagsList v-if="pool !== undefined && pool.tags.length > 0">
+                <UiTag v-for="tag in pool.tags" :key="tag" accent="info" variant="primary">{{ tag }}</UiTag>
+              </UiTagsList>
+            </template>
+            <template v-if="pool !== undefined && pool.tags.length > 0" #addons>
+              <VtsCopyButton :value="pool.tags.join(', ')" />
+            </template>
+          </VtsCardRowKeyValue>
+        </div>
+      </UiCard>
+      <UiAlert v-else accent="danger">
+        {{ t('connection-failed') }}
+        <template #description>
+          {{ t('unable-to-connect-to-the-pool') }}
+        </template>
+      </UiAlert>
+      <UiCard class="card-container">
+        <UiCardTitle class="typo-body-bold text-ellipsis">
+          {{ t('connection') }}
+        </UiCardTitle>
+        <!-- status -->
+        <VtsCardRowKeyValue>
+          <template #key>{{ t('status') }}</template>
+          <template #value>
+            <UiInfo :accent="connectionStatus.accent">
+              {{ connectionStatus.text }}
+            </UiInfo>
+          </template>
+        </VtsCardRowKeyValue>
+        <!-- primary-host -->
+        <VtsCardRowKeyValue>
+          <template #key>{{ t('master') }}</template>
+          <template #value>
+            <UiLink v-if="primaryHost" :icon="faServer" size="small" :to="`/host/${primaryHost.id}/`">
```

Comment:
Just to keep consistency with the line below and the rest of the code:
```suggestion
            <UiLink v-if="primaryHost !== undefined" :icon="faServer" size="small" :to="`/host/${primaryHost.id}/`">
```